internal/app/oauth/storage.go

```diff
@@ -0,0 +1,337 @@
+package oauth
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"errors"
+	"math/big"
+	mRand "math/rand"
```

Comment:
```suggestion
	mrand "math/rand"
```

Uppercase is not used in package name